### DISCUSSION TITLE
Errors when installing from source.

### DISCUSSION THREAD

**Discussion by meton-robean** [milvus-io/milvus#4559] [Q&A and General discussion]
I have install milvus from source code following the [guide of  its github  README](https://github.com/milvus-io/milvus/blob/master/INSTALL.md) , but I met an errors when i ran 
```
./build.sh -g -t Release
```

And the ERROR is shown as follows:
```
-- Building ZLIB-v1.2.11 from source
-- Building OPENTRACING-v1.5.1 from source
-- Building FIU-1.00 from source
-- Building oatpp-1.0.1 from source
-- Building Milvus GPU version
CMake Error at CMakeLists.txt:158 (add_compile_definitions):
  Unknown CMake command "add_compile_definitions".


-- Configuring incomplete, errors occurred!
See also "/home/cmt/milvus/core/cmake_build/CMakeFiles/CMakeOutput.log".
See also "/home/cmt/milvus/core/cmake_build/CMakeFiles/CMakeError.log".
make: *** No rule to make target 'install'.  Stop.

```

It seems that it is a error of CMAKE version, but my cmake version is 3.5.1, which met the minimum_required in [CMakeList.txt](https://github.com/milvus-io/milvus/blob/master/core/CMakeLists.txt)
```
cmake_minimum_required( VERSION 3.14 )
```

**Comment by yhmo** [comment]
3.5.x is older than 3.14.x. Please make sure your cmake version higher than 3.14.0
For example, my cmake version is 3.14.4
